packages/block-editor/src/default-editor-styles.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Default editor styles.
+ *
+ * These styles are shown if a theme does not register its own editor style,
+ * a theme.json file, or has toggled off "Use theme styles" in preferences.
+ */
+
+body {
+	font-family: $default-font;
+	font-size: 18px;
+	line-height: 1.5;
+	--wp--style--block-gap: 2em;
+}
+
+p {
+	line-height: 2;
```

Comment:
```suggestion
	line-height: 1.8;
```

The line height feels a tiny bit too tall to me — this seems more readable: 

Before|After
---|---
<img width="441" alt="Screen Shot 2021-09-01 at 7 40 44 AM" src="https://user-images.githubusercontent.com/1202812/131665295-eca4a6f1-48a7-48b6-9504-89cb91f79b41.png">|<img width="432" alt="Screen Shot 2021-09-01 at 7 40 29 AM" src="https://user-images.githubusercontent.com/1202812/131665302-c6bdfa10-da97-4bc9-8d56-aae1ca91f212.png">